contextualSpellCheck/contextualSpellCheck.py

```diff
@@ -379,18 +379,21 @@ def candidate_ranking(self, doc, misspellings_dict):
         response = {}
         #         doc = self.nlp(query)
         for misspell in misspellings_dict:
-            # Init least_edit distance
-            least_edit_dist = self.max_edit_dist
-
             if self.debug:
                 print(
                     "misspellings_dict[misspell]", misspellings_dict[misspell]
                 )
-            for candidate in misspellings_dict[misspell]:
-                edit_dist = editdistance.eval(misspell.text, candidate)
-                if edit_dist < least_edit_dist:
-                    least_edit_dist = edit_dist
-                    response[misspell] = candidate
+
+            if self.max_edit_dist:
```

Comment:
This does not feel correct to me. `max_edit_dist` sounds like the maximum edit distance which is not allowed -> `edit distance <= max_edit_dist`. However it appears to mean `edit distance < max_edit_dist`. So for `max_edit_dist = 0` there can never be any result, since there can never be an edit distance below 0.